spec/src/main/asciidoc/core/spi_lite.asciidoc

```diff
@@ -1,12 +1,386 @@
 [[spi_lite]]
 
-== Build time extensions
+== Build compatible extensions
 
-// TODO
-[NOTE]
-====
-TBD
-====
+A build compatible extension may integrate with the container during deployment time, as defined in <<initialization>>.
 
-// Must mention that Build Compatible Extensions do not have access to the CDI container,
-// as there may not be one running! Calling `CDI.current()` inside them results in non-portable behavior.
+Comprehensive specification of build compatible extensions is available in the javadoc.
```

Comment:
No other part of spec refers to javadoc, I wouldn't do it here either. The overview you wrote is IMO informative enough.

Reply:
There's actually _one_ other part of the spec that does refer to a javadoc :-) But if the javadoc text is implicitly assumed to be part of the spec, then sure.